app/views/internal/users/edit.html.erb

```diff
@@ -34,7 +34,9 @@
     <%= form_for(@user, url: user_status_internal_user_path(@user), html: { class: "mt-3" }) do |f| %>
       <div class="form-group">
         <%= f.label "Select new user status", class: "mr-3" %>
-        <%= f.select(:user_status, options_for_select(["Warn", "Comment Ban", "Ban", "Regular Member", "Trusted", "Pro"]), include_blank: true) %>
+        <% roll_select = ["Warn", "Comment Ban", "Ban", "Regular Member", "Trusted", "Pro"] %>
```

Comment:
How about moving this into `app/lib/constants.rb` as `ROLES` and `SPECIAL_ROLES`, then changing this to 

```ruby
roll_select = current_user.has_role?(:super_admin) ? ROLES + SPECIAL_ROLES : ROLES
```

Reply:
Good call